scrapers/la/__init__.py

```diff
@@ -256,6 +256,15 @@ class Louisiana(State):
             "name": "2024 Second Extraordinary Session",
             "start_date": "2024-02-19",
             "end_date": "2024-03-06",
+            "active": False,
+        },
+        {
+            "_scraped_name": "2024 Third Extraordinary Session",
+            "classification": "special",
+            "identifier": "2024s3",
+            "name": "2024 Third Extraordinary Session",
+            "start_date": "2024-11-01",
+            "end_date": "2024-11-29",
             "active": True,
```

Comment:
@jessemortenson when there's no bills yet for a session, add it with "active" set to False. States occasionally drop test bills, broken bills, the wrong session's, etc in the first day of a session, so somebody should do a quick manual review before flipping to active so the scrapers don't pick up a bunch of junk that needs to be deleted.

Reply:
Cool, thanks for the reminder here - I'm a little rusty and always appreciate the feedback :)